addons/vscode/src/extension.ts

```diff
@@ -81,17 +82,17 @@ export async function getCliPath(extensionPath?: string): Promise<string> {
 	state.configPath = configPath;
 
 	const executableExists = (path: string) => {
```

Comment:
Nitpicking time. Since we are checking version in the function, we'd better to rename it from `executableExists`, to e.g. `isValidExecutable`.

Reply:
renamed